dist/3rdpartylicenses.txt

```diff
@@ -1,92 +0,0 @@
-core-js@2.5.1
```

Comment:
Don't rebuild dist files, Steve or I will generate dist files. This makes merging, reviewing and ensuring code integrity easier.